Gems/ROS2/Code/Source/RobotImporter/URDF/CollidersMaker.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "UrdfParser.h"
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/parallel/atomic.h>
+#include <AzCore/std/parallel/mutex.h>
+#include <AzCore/std/parallel/thread.h>
+#include <AzCore/std/smart_ptr/make_shared.h>
+#include <AzCore/std/smart_ptr/shared_ptr.h>
+#include <AzFramework/Physics/Material/PhysicsMaterialId.h>
+#include <AzFramework/Physics/Material/PhysicsMaterialManager.h>
+#include <RobotImporter/Utils/SourceAssetsStorage.h>
+
+namespace ROS2
+{
+    using BuildReadyCallback = AZStd::function<void()>;
+
+    //! Populates a given entity with all the contents of the <collider> tag in robot description.
+    class CollidersMaker
+    {
+    public:
+        //! Construct the class based on URDF asset mapping.
+        //! @param urdfAssetsMapping a prepared mapping of Assets used by the source URDF.
+        CollidersMaker(const AZStd::shared_ptr<Utils::UrdfAssetMap>& urdfAssetsMapping);
+
+        //! Deleted copy constructor.
```

Comment:
No need for comment, is self-evident. Or state the purpose of deleting the copy constructor
```suggestion
        //! Prevent copying of existing CollidersMaker
```

Reply:
done